index.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+
+    <h1 id="title">freeCodeCamp Survey Form</h1>
+    <p id="description">Thank you for taking the time to help us improve the platform</p>
+    <div class="form">
```

Comment:
Is this how you work? Uh, it's so funny! make sure to use the `<main>` semantic tag in here. It's not okay. It's bad very bad.

Reply:
- [ ]  Please communicate politely with the student make suggestions instead of ordering, and give them more details about why you asked them to bring changes to the project.
- [ ]  You can say " Please make sure to use  `main` semantic tag here to wrap your code ".